src/format/opamFile.ml

```diff
@@ -1170,12 +1194,13 @@ end
 module ConfigSyntax = struct
 
   let internal = "config"
-  (* This version is used as a marker for the whole opam root, so it's not
-     strictly speaking the actual format of the config file *)
   let format_version = OpamVersion.of_string "2.1"
+  let file_format_version = OpamVersion.of_string "2.0"
+  let root_version = OpamVersion.of_string "2.1.0~rc"
```

Comment:
I didn't set it up to current in case we have several updates in a single release., we want to be able to have `2.2.0~alpha.2` for ex

Reply:
That's a very good point! In which case, does it just want to drop the `.0` (so be `2.1~rc`) - we might do `2.1~rc2`, etc. but I think we can commit that we'd _never_ do `2.1.1`?

Reply:
it should never happen, yes! The need to have a format change in minor version is in case of a bug fix that need this change, but I see it as another argument to do more frequent releases :D

Reply:
Updated!